src/elm/expression.ts

```diff
@@ -39,7 +39,7 @@ export class Expression {
 
   async execArgs(ctx: Context) {
     if (this.args != null) {
-      // TODO (MATT): check this
+      // TODO (MATT) ALERT: check this. doesn't work at all. sad
```

Comment:
thanks for leaving this in hehehe

Reply:
Didn't even notice this. Must have been from your cherry picked commit.